cmd/execution_builder.go

```diff
@@ -620,14 +621,26 @@ func (exeNode *ExecutionNode) LoadExecutionDataGetter(node *NodeConfig) error {
 	return nil
 }
 
+func openChunkDataPackDB(dbPath string) (*badgerDB.DB, error) {
+	db, err := badgerDB.Open(badgerDB.LSMOnlyOptions(dbPath))
```

Comment:
the public badgerdb initialization uses some more options. Is this all that's needed for this usecase?